.circleci/build_autoinst.sh

```diff
@@ -18,12 +18,12 @@
 set -ex
 
 destdir=${1:-../os-autoinst}
-sha=${2}
+sha=${2:-$(cat ./autoinst.sha)}
```

Comment:
should that be `sha=${2:-$(cat .circleci/autoinst.sha)}` ? Not sure if the working directory is in `.circleci` or out of `.circleci/`, I see the config.yml means out it.

Reply:
I changed it now to not specify it in `.circleci/config.yml` and assume we're at the toplevel.